src/modules/LanItems.rb

```diff
@@ -488,7 +488,7 @@ def ReplaceItemUdev(replace_key, new_key, new_val)
       # =    for assignment
       # ==   for equality checks
       operator = new_key == "NAME" ? "=" : "=="
-      current_rule = getUdevFallback
+      current_rule = GetItemUdevRule(@current)
```

Comment:
Just obtain the current rule without generating a new one if not exist should fix it.